archivebox/config.py

```diff
@@ -696,6 +748,28 @@ def load_config(defaults: ConfigDefaultDict,
 
 #     with open(os.path.join(config['OUTPUT_DIR'], CONFIG_FILENAME), 'w+') as f:
 
+def parse_tag_name(v):
+    """parses a version tag string formatted like 'vx.x.x'"""
+    v = re.sub(r"\+.*$", "", v) # in case version string ends with '+editable'
+    parts = re.sub(r"^v", "", v).split(".")
+    return [int(p) for p in parts]
+
+
+def compare_versions(v1, v2):
```

Comment:
We shouldn't need to do any manual version comparisons as the GitHub API should return the releases in order. Just use their position in the release list as the sort key. This will remain working if we move away from semver to date-based versions later.

Reply:
The only thing I thought this is needed for is to link to the release page for the currently installed version. I thought a comparison is necessary in case the installed version name doesn't match any of the version names in the releases (like if the installed version is "0.7.0"). I decided to link to the first version in the list of releases that's older than the currently installed, but maybe we shouldn't link to anything in this case?

Reply:
I removed that function, now just making the parsing function return a tuple which can be compared directly (I think this is basically what you suggested in another comment). If we change to date-based versions I think the logic can stay the same.